geps/gep-1897/index.md

```diff
@@ -22,18 +22,15 @@ the service or backend owner wants to validate the clients connecting to it, two
 1. The solution must satisfy the following use case: the backend pod has its own
 certificate and the gateway implementation client needs to know how to connect to the
 backend pod. (Use case #4 in [Gateway API TLS Use Cases](#references))
-2. In terms of the Gateway API personas, only the application developer persona applies in this
-solution. The application developer should control the gateway to backend TLS settings,
-not the cluster operator, as requiring a cluster operator to manage certificate renewals
-and revocations would be extremely cumbersome.
-3. The solution should consider client certificate settings used in the TLS handshake **from
-Gateway to backend**, such as server name indication, trusted certificates,
-and CA certificates.
+2. Both the application developer persona, and gateway operator persona will have control over TLS settings.
+3. The solution should consider client TLS settings used in the TLS handshake **from
+Gateway to backend**, such as server name indication and trusted CA certificates.
+4. Both Gateway and Mesh use cases will be supported.
```

Comment:
If we already have an experimental Gateway-only implementation, we cannot require that they add Mesh use cases in order to claim a Standard implementation.  I don't think that's made clear here.

Reply:
This is definitely not the intent! Gateway API currently has different profiles for Mesh and Gateways, so all of the existing non-Mesh implementations do not run the 'Mesh' profile, and are not impacted in any way by any mesh features.

Reply:
```suggestion
4. Both Gateway and Mesh use cases may be supported, depending on the implementation, and will be covered by features in each case.
```